srv/salt/_modules/osd.py

```diff
@@ -178,7 +178,7 @@ def _children():
     """
     Returns the OSDs assigned to this node according to Ceph
     """
-    result = json.loads(tree())
+    result = tree_from_any()
```

Comment:
NOTE: this is the only place in the entire DeepSea codebase I could find where `osd.tree` was being called. If there are any other calls, this PR will break them.